extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -383,7 +391,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
-	public async openNotebookFolder(): Promise<void> {
+	public async openNotebookFolder(urlToOpen?: string, showPreview?: boolean): Promise<void> {
```

Comment:
Are there any issues with loading multiple notebooks at once when using these options?

Reply:
I haven't seen any issues loading multiple notebooks but the highlight isn't working as expected. I'll create a separate issue to track it.